state-engine/src/main/java/transaction/TxnManager.java

```diff
@@ -1,5 +1,6 @@
 package transaction;
 
+import org.apache.zookeeper.txn.Txn;
```

Comment:
Again...why we need to use zookeeper here?

Reply:
Already delete